sharding-core/sharding-core-parse/sharding-core-parse-mysql/src/main/antlr4/imports/mysql/MySQLKeyword.g4

```diff
@@ -1511,3 +1511,7 @@ UNIX_TIMESTAMP
 LOWER
     : L O W E R
     ;
+
+UPPER
+    : U P P E R
+    ;
```

Comment:
please add a blank line in the end of file

Reply:
Done